Split up line 204 which was greater than 72 characters

asoloway64 opened the pull request.

Commit:
Update draft-ietf-core-conditional-attributes.md

cabo commented on draft-ietf-core-conditional-attributes.md:
```diff
@@ -201,7 +201,9 @@ bool notifiable( Resource * r ) {
 #define ST_EX ( abs( r->v - r->last_rep_v ) >= r->st )
-#define IN_BAND ( ( r->gt <= r->v && r->v <= r->lt ) || ( r->lt <= r->gt && r->gt <= r->v ) || ( r->v <= r->lt && r->lt <= r->gt ) )
+#define IN_BAND ( ( r->gt <= r->v && r->v <= r->lt ) ||
+                  ( r->lt <= r->gt && r->gt <= r->v ) ||
```
C requires backslash-escaping continuation lines of #defines...

bsilverajan replied:
Very true, that was my mistake!

bsilverajan reviewed (changes requested):
Thanks Alan and Carsten. 

Alan could you please amend the change to include ~~leading backslashes to lines 205 and 206~~  trailing backslashes to lines 204 and 205?

(Edited based on Carsten's feedback)

cabo commented:
Actually, I think you need trailing backslashes on 204 and 205, but I haven't used continuation lines on #define for quite some time.

bsilverajan commented:
> Actually, I think you need trailing backslashes on 204 and 205, but I haven't used continuation lines on #define for quite some time.

Very true, that was my mistake!

Commit:
Added backslashes to continued line in C code

cabo committed:
Update draft-ietf-core-conditional-attributes.md

I can compile the snippet (with some prelude) after changing the double backslashes into single ones.

asoloway64 commented:
Markdown uses backslash for control characters, so my understanding was that you needed the double-backslash to get a backslash displayed.

bsilverajan commented:
> Markdown uses backslash for control characters, so my understanding was that you needed the double-backslash to get a backslash displayed.

I ran both versions through xml2rfc and it seems just a single backslash suffices. So I think we can merge the PR with Carsten's changes. 

Thank you both!
